docs/modules/ROOT/partials/proc-registry-sql-backup.adoc

```diff
@@ -1,7 +1,7 @@
 [id=registry-sql-backup]
 = Backing up {registry} PostgreSQL storage
```

Comment:
@smccarthy-ie the procedure described in this section still works, because it's 100% based on postgresql, but as you know we now have the import/export api , so maybe we could add a link here to the import/export api guide or something that mentions that we have an API useful for backups

Reply:
@famartinrh Good point. @jsenko Suggest something like the following added to this file and proc-registry-sql-restore.adoc: 

Additional resources

* For details on client authentication, see the link:https://www.postgresql.org/docs/12/client-authentication.html[PostgreSQL documentation].
ifdef::apicurio-registry[]
* For details on importing and exporting registry content, see link:https://www.apicur.io/registry/docs/apicurio-registry/2.0.0.Final/getting-started/assembly-managing-registry-artifacts-api.html[Managing Apicurio Registry content using the REST API].
endif::[]
ifdef::service-registry[]
* For details on importing and exporting registry content, see link:https://access.redhat.com/documentation/en-us/red_hat_integration/{version}/html/service_registry_user_guide/managing-registry-artifacts-api[Managing Apicurio Registry content using the REST API].
endif::[]